k8-util/helm/fluvio-sys/templates/crd_managed_connector.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
bump up chart version

Reply:
Hmm, which part? I get errors if I the `apiVersion` field and the `versions.name` field.